scalableminds/webknossos
# Allow to select user scripts in the Add Script modal

See title.
I would love to add syntax highlighting to that textarea, but haven't found a simple solution. Not sure whether it's used that much and we should include another clunky lib to accomplish this.

### Mailable description of changes:
 - Added the possibility to select a pre-defined user script in the "Add scripts" modal

### Steps to test:
- Add a user script in the administration interface. Check whether you can select it in the Add scripts modal.
- Modify the script in the modal and try to select another script - a confirm dialog should appear
- Check whether tasks with attached scripts still execute the script successfully

### Issues:
- fixes #1918

------
